AngularApp/src/app/diagnostic-data/components/star-rating-feedback/star-rating-feedback.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'star-rating-feedback',
+  templateUrl: './star-rating-feedback.component.html',
+  styleUrls: ['./star-rating-feedback.component.css']
+})
+export class StarRatingFeedbackComponent implements OnInit {
+  starList: boolean[] = [false, false, false, false, false];
+  rating: number;
+  isSubmitted: boolean = false;
+  isRated: boolean = false;
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
remove ngOnInit() and 'implements OnInit' if it is not using it

Reply:
Sure get it removed here.